src/common/buildApp.ts

```diff
@@ -0,0 +1,20 @@
+import App from '@/components/App.vue';
+import store from '@/store';
+import TermboxRequest from '@/common/TermboxRequest';
+import { NS_ENTITY } from '@/store/namespaces';
+import { ENTITY_INIT } from '@/store/entity/mutationTypes';
+import Vue from 'vue';
+
+export default ( req: TermboxRequest ) => {
```

Comment:
I think the shared TermboxRequest is a key concept in going forward (with the concrete class being to superior concept to the interface) and staying sane.